dtfs-central-api/api-tests/v1/utilisation-reports/get-utilisation-report.api-test.ts

```diff
@@ -0,0 +1,63 @@
+import { Response } from 'supertest';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { AzureFileInfo, IsoDateTimeStamp, ReportPeriod, UtilisationReportEntity, UtilisationReportReconciliationStatus } from '@ukef/dtfs2-common';
+import app from '../../../src/createApp';
+import apiModule from '../../api';
+import { anUploadedUtilisationReportEntity } from '../../mocks/entities/utilisation-report-entity';
+
+const api = apiModule(app);
+const getUrl = (id: string) => `/v1/utilisation-reports/${id}`;
+
+const givenUploadedReport = async (): Promise<string> => {
```

Comment:
Is this naming trying to follow the like given when then approach and why it's framed like this? A pattern like this makes sense, wondering if it's slightly less clear that it's uploading a report and returning one if you don't know the pattern - thoughts?

Reply:
Yeah, the idea behind it is to name the function in a way that makes the test as readable as possible, leaving out the technical details. However I see your point, and think it is perfectly fine if we would prefer more of the detail in the test to not diverge too far from current tests, especially as we are returning the id which is non-obvious unless it is a pattern you know and are familiar with.

Given this is only used once I will perhaps just move it into the test rather than write a more descriptive function name and then on tests where there is reusability advantage to a helper function like this go with a slightly more descriptive name?

Reply:
Yeah sounds good to me :)